packages/backend/src/scripts/kafka_batch_publisher.ts

```diff
@@ -427,7 +427,7 @@ const publishCourseAnswers = async (
       }
 
       if (useUserPointsBatch) {
-        return
+        continue
       }
```

Comment:
Redundant?

Reply:
Not redundant, we don't want to send messages for all exercises (as happens if `userUserPointsBatch` == false), but we want to continue the loop.

Reply:
I didn't notice the loop there. All good then!